include/config_reader/macros.h

```diff
@@ -37,60 +37,62 @@ namespace config_reader {
 // Define macros for creating new config vars
 
 #define CONFIG_INT(name, key)                                                  \
-  const auto& MAKE_NAME(name) =                                                \
+  const int& MAKE_NAME(name) =                                                 \
       ::config_reader::InitVar<int, ::config_reader::config_types::ConfigInt>( \
           key)
-#define CONFIG_UINT(name, key)                            \
-  const auto& MAKE_NAME(name) = ::config_reader::InitVar< \
+#define CONFIG_UINT(name, key)                                      \
+  const(unsigned int) & MAKE_NAME(name) = ::config_reader::InitVar< \
```

Comment:
Why is `(unsigned int)` parenthesized?

Reply:
Because it is a type using two seperate words. I can remove the paranethsis but in my mind it made more sense.